### ISSUE TITLE
Issue with Concordia diagrams after removal of RM analysis

### ISSUE THREAD

**Issue by AllenKennedy** [CIRDLES/Squid#618]
Removal of the BR5.7 analysis from the list of RM analyses using right click under Data results in the RM points on the Concordia diagram shifting to the right, and no longer plotting near the Concordia. Shouldn't the RM analyses always plot at or very near the true RM isotopic composition on the Concordia? This offset also occurs for the Concordia of the unknowns. I suspect I am doing something wrong, but cannot figure out what. 
 
[E3721st RM remove test start.squid.zip](https://github.com/CIRDLES/Squid/files/6432893/E3721st.RM.remove.test.start.squid.zip)

<img width="1249" alt="Screen Shot 2021-05-06 at 3 49 30 pm" src="https://user-images.githubusercontent.com/33318855/117262909-516dd280-ae84-11eb-92d6-3d34b93d8593.png">

<img width="1248" alt="Screen Shot 2021-05-06 at 3 51 14 pm" src="https://user-images.githubusercontent.com/33318855/117262961-5e8ac180-ae84-11eb-9ca0-2ac230b942e5.png">

**Comment by NicoleRayner** [comment]
This is weird, I can't explain it but I do note a couple of things.  If you exclude analysis BR5.7 (rather than remove it from the xml file) everything works fine.  But also, that if you remove the analysis when you are interpolating to mid-spot time (rather than spot average) you don't have this behaviour.

![image](https://user-images.githubusercontent.com/40035242/117311457-32eee380-ae52-11eb-8f91-1a03c0eb0fb3.png)

![image](https://user-images.githubusercontent.com/40035242/117311640-5b76dd80-ae52-11eb-892a-180cef0eaaaa.png)

**Comment by bowring** [comment]
@AllenKennedy - great catch! and  @NicoleRayner - great discovery.  I can confirm that merely switching back to linear regression and then returning to spot average fixes the issue.  This process yields the same results as creating a new project from a data file missing BR5.7. @AllenKennedy please confirm. Stay tuned for updates.

**Comment by NicoleRayner** [comment]
I can make this happen on any file (e.g. the demo one) - the shift off concordia is just more subtle.

**Comment by bowring** [comment]
Agreed - but the switch mentioned above fixes it temporarily until I get to the bottom of it - do you concur?

**Comment by NicoleRayner** [comment]
I concur.  And I just checked that if you 1) open a xml file, 2) remove an analysis is Data/Manage spots, THEN 3) load the task there is no problem.  Its if you load the xml, followed by loading the task and THEN remove a spot that you get problems (that are resolved by toggling between linear regression and spot average).  So removing spots AFTER loading the task is bad news.
